test/sdp.jl

```diff
@@ -395,17 +395,8 @@ context("With solver $(typeof(solver))") do
         @defVar(m, u[1:d])
         @defVar(m, μ[1:d])
 
-        @defVar(m, t1 >= 0)
-        @defVar(m, L1[1:d])
-        @addConstraint(m, L1 .== (μ-μhat))
-        @addConstraint(m, sum{L1[i]^2, i=1:d} <= t1^2)
-        @addConstraint(m, t1 <= Γ1(𝛿/2,N))
```

Comment:
Could we leave these in as a separate test? Always good to test multiple ways of doing the same thing